script.js

```diff
@@ -0,0 +1,68 @@
+let collectionBooks = [];
+const inputsObject = {};
+
+const title = document.querySelector('#input-title');
+const author = document.querySelector('#input-author');
+
+let position = -1;
+
+class NewBook {
+  constructor(title, author, index) {
+    this.title = title;
+    this.author = author;
+    this.index = index;
+  }
+}
+
+function printf(collectionBooks) {
+  document.getElementById('container-book').innerHTML = collectionBooks.map((items) => `<div id="cards">
+    <h3>${items.title}</h3>
+    <p>${items.author}</p>
+    <button onclick="deletInfo(${items.index})">Remove</button>
+    <hr>
+    </div>`).join('');
+}
+
+// check with every letter clicked
+function saveInfo() {
+  const newtitle = document.querySelector('#input-title').value;
+  const newauthor = document.querySelector('#input-author').value;
+
+  // Counter to define the index in the colections books
+  position += 1;
+
+  // If the title or the author is empty, then don't add the book
+  if (newtitle === '' || newauthor === '') {
+    return null;
+  }
+  // Create a new Book
+  const bookInfo = new NewBook(newtitle, newauthor, position);
+  collectionBooks.push(bookInfo);
+
+  // Print in the html file
+  printf(collectionBooks);
+  return collectionBooks;
+}
+
+document.querySelector('#bookForm').addEventListener('submit', () => {
+  inputsObject.title = title.value;
+  inputsObject.author = author.value;
+  localStorage.setItem(title.value, JSON.stringify(inputsObject));
+});
```

Comment:
- Please make sure data is preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md), reload the page manually to check this after adding a book and removing a book.

Reply:
Hi @rotimiazeez, isn't that what I'm doing here?
![image](https://user-images.githubusercontent.com/97189760/182304736-b7c4524e-eae3-40cf-b4c8-0dc135ca95fa.png)

Reply:
@angeluray My sincere apologies it was a mix up from my end Kindly submit again and it will be approved 🙏🏽

Reply:
@rotimiazeez It's ok. I'll submit it again then.